app/Keycloak/Repositories/KeycloakClientRepository.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Keycloak\Repositories;
+
+interface KeycloakClientRepository
```

Comment:
This misses all the methods that need to be implemented inside the eloquent concrete implementation.